ch_integration_tests/tests/integration.rs

```diff
@@ -718,4 +718,96 @@ mod tests {
 
         assert!(r.is_ok());
     }
+
+    #[test]
+    fn test_bridge_networking() {
+        use std::fs;
+        use std::io::Write;
+        use std::process::Command;
+        cleanup_libvirt_state();
+
+        //configure host IP  since we cant do it from XML for bridge network.
+        Command::new("sh")
+                .arg("-c")
+                .arg("sudo ip addr add 192.168.1.1/24 dev mybr0 ; sudo ip addr add 192.168.2.1/24 dev mybr0")
+                .spawn()
+                .expect("Failed to set IP address.");
+
+        let mut guest_name: [String; 2] = ["guest1".to_string(), "guest2".to_string()];
+        let mut guest_ip: [String; 2] = ["ip1".to_string(), "ip2".to_string()];
```

Comment:
I recommend setting these to `Default::default()` instead (handy thing setting lots of initial values).